.eslintrc

```diff
@@ -16,6 +17,13 @@
     "flowtype/require-return-type": ["error", "always", {"excludeArrowFunctions": true}],
     "require-await": "error",
     "no-return-await": "error",
-    "sort-keys": "off"
+    "sort-keys": "off",
+    "prettier/prettier": ["error", {
+      "singleQuote": true,
+      "trailingComma": "all",
+      "bracketSpacing": false,
+      "printWidth": 120,
+      "parser": "flow"
+    }]
   }
 }
```

Comment:
@kaylieEB I tried the cli tool https://github.com/prettier/eslint-config-prettier#cli-helper-tool but there are some problems: every yarn command put `yarn run v<version>` on top of the result of each command.. when I try to pipe with `eslint-config-prettier-check` it does not work.
Is it ok for you if we open an issue and we work on that in another pull request?